### ISSUE TITLE
EOF in aws_lb_listener when ALB is configured with the new "fixed response" option (via AWS console)

### ISSUE THREAD

**Issue by tam116** [terraform-providers/terraform-provider-aws#5357]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version
0.11.7
aws = 1.29,1.27,1.28, and 1.22

### Affected Resource(s)
aws_lb_listener_rule

### Terraform Configuration Files

```hcl
resource "aws_lb_listener_rule" "LoadBalancerListener" {
  listener_arn = "${var.ListenerArn}"

  action {
    type             = "forward"
    target_group_arn = "${var.TargetGroupArn}"
  }

  condition {
    field  = "host-header"
    values = "${var.values}"
  }

  lifecycle {
    create_before_destroy = true
  }
}
```

### Debug Output
https://gist.github.com/tam116/31d807604ca39d28f8651a96bec82d82

### Panic Output
https://gist.github.com/tam116/31d807604ca39d28f8651a96bec82d82

### Expected Behavior
Terraform should apply the config. It should ignore the "fixed response" listener rule configured on the ALB when that rule is not managed by terraform. If the rule is managed it should handle setting the rule back to a "forward" (or any other supported listener rule type)

### Actual Behavior
Terraform plan fails with EOF errors.

### Steps to Reproduce
Manually configure the default ALB rule to be a "fixed response" listener rule via web console of aws cli. 
Use terraform to manage a different "forward" rule on the same ALB.

### Important Factoids
I did not test against other rule types like Authenticate or Redirect nor setting a non-default rule to fixed response,

### References
Details about the new ALB listener rule options: https://aws.amazon.com/about-aws/whats-new/2018/07/elastic-load-balancing-announces-support-for-redirects-and-fixed-responses-for-application-load-balancer/

**Comment by junkangli** [comment]
I am facing the same problem when manually configuring the default action to "Redirect".

**Comment by bflad** [comment]
I'm working on a fix for this, I should be able to submit the pull request shortly.

**Comment by bflad** [comment]
Crash fix pull request submitted: #5367

**Comment by bflad** [comment]
The fix for this crash has been merged into master and will release with version 1.30.0 of the AWS provider, likely middle of next week.

**Comment by bflad** [comment]
This has been released in [version 1.30.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1300-august-02-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
